packages/blade-svelte/src/components/Link/BaseLink/baseLink.css

```diff
@@ -0,0 +1,43 @@
+.base-link {
```

Comment:
is font-family not defined in our react BaseLink? can you check how fonts are working in react and if we can get that implemented in svelte as well?